src/languages/es.js

```diff
@@ -764,8 +764,14 @@ export default {
         reimburse: {
             captureReceipts: 'Captura recibos',
             fastReimbursementsHappyMembers: '¡Reembolsos rápidos = miembros felices!',
+            kilometers: '',
+            miles: '',
             viewAllReceipts: 'Ver todos los recibos',
             reimburseReceipts: 'Reembolsar recibos',
+            trackDistance: '',
+            trackDistanceCopy: '',
+            trackDistanceRate: '',
+            trackDistanceUnit: '',
```

Comment:
This is wrong, you need to complete the translations for this language. 
Since it was not done, it generated a few alerts. I am going to post them in the original issue so you can get them fixed @marktoman.
@deetergp you should've not merged this

Reply:
Lo siento! 😬  Not sure how I missed that but will get some translations to Mark Toman, so he can add them.

Reply:
It looks like the translations have already been added in `main`
![Screen Shot 2022-02-17 at 9 22 00 AM](https://user-images.githubusercontent.com/766197/154536105-f12304ec-db7b-4824-847b-c6683fd306dc.png)
 
Is that all that was missing to fix those BugBot issues?

Reply:
Oh did not see that, I just saw the bugbot issues and assumed it was still happening. Curious where were they added?
And yes, that's all we need to do to fix the issues.